django_evaluation/settings/local.py

```diff
@@ -77,6 +77,23 @@ def _get_logo(logo_file, project_root):
     return f"/static/img/{logo_file.name}"
 
 
+def _read_secret(port: int = 5002, key: str = "email") -> dict[str, str]:
```

Comment:
@antarcticrainforest I have no clue about those changes and where they come from. Do we need this? Apparently it is not in use.